jest.config.js

```diff
@@ -10,10 +10,10 @@ module.exports = {
   maxWorkers: 1,
   coverageThreshold: {
     "./src/coinbase/**": {
-      branches: 77,
-      functions: 90,
-      statements: 85,
-      lines: 87,
+      branches: 70,
```

Comment:
this is a significant reduction. please only make adjustments required after your newly added classes have required coverage

Reply:
hey @howard-at-cb are you seeing any errors that required you to change this? on your branch i just tried keeping the original thresholds and it seems to pass locally when i run the tests

Reply:
modified accordingly. Without this change I am seeing below errors '
```
Jest: "/Users/howardxie/src/coinbase-sdk-nodejs/src/coinbase/wallet.ts" coverage threshold for lines (87%) not met: 86.84%
Jest: "/Users/howardxie/src/coinbase-sdk-nodejs/src/coinbase/wallet.ts" coverage threshold for functions (90%) not met: 88.46%
```